bugwarrior/services/jira.py

```diff
@@ -223,7 +223,9 @@ def get_priority(self):
             value = value['name']
         except (TypeError, ):
             value = str(value)
-        return self.PRIORITY_MAP.get(value, self.origin['default_priority'])
+        # priority.name format: "1 - Critical"
+        map_key = value.strip().split()[-1]
```

Comment:
I'm worried this might screw up some other people if they have custom priorities set to something like "Very Important" which would split their string into just "Important".

Can you think of a way to make this formatting configurable?

Reply:
This module could query the priority list and map accordingly. That will require a little bit more work. In the meantime, priority imports don't work for the default behavior of several JIRA versions. This module has a fixed mapping, so doesn't support custom priorities anyway.

Reply:
Ah, fair.  I'm game to merge this as is and take on custom priorities at a later time.  Thanks!